rotkehlchen/chain/ethereum/manager.py

```diff
@@ -1195,3 +1195,23 @@ def get_basic_contract_info(self, address: ChecksumEthAddress) -> Dict[str, Any]
 
         self.contract_info_cache[address] = info
         return info
+
+    def _get_basic_contract_info_fallback(
+            self,
+            output: List[Tuple[bool, bytes]],
+            properties: Tuple[str, str, str],
```

Comment:
Maybe we can move the tuple to a constant like `ERC20_INFO_METHODS` and have its own type so we can use it here also